src/main/java/hudson/plugins/cppncss/parser/StatisticsResult.java

```diff
@@ -51,6 +59,11 @@ public void clear() {
 		 fileResults.clear();
 	}
 
+	/**
+	 * @deprecated  Should not be used.

```

Comment:
Did you check whether these accessors are used from Jelly?

Reply:
yes, no usages in Jelly